addon/package.json

```diff
@@ -40,7 +40,7 @@
     "@babel/core": "^7.16.12",
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-proposal-decorators": "^7.16.7",
-    "@embroider/addon-dev": "^1.0.0",
+    "@embroider/addon-dev": "^4.1.0",
```

Comment:
I updated to the most recent addon-dev version, as v1.0 created incorrect dist files.